README.md

```diff
@@ -1,13 +1,80 @@
-# ACK service controller for AWS Certificate Manager Private Certificate Authority
+# ACK service controller for AWS Private Certificate Authority
 
 This repository contains source code for the AWS Controllers for Kubernetes
-(ACK) service controller for ACM-PCA.
+(ACK) service controller for AWS Private Certificate Authority.
 
 Please [log issues][ack-issues] and feedback on the main AWS Controllers for
 Kubernetes Github project.
 
 [ack-issues]: https://github.com/aws/aws-controllers-k8s/issues
 
+## Resources Supported
+The ACK service controller for AWS Private Certificate Authority supports the following resources:
+- CertificateAuthority
+- Certificate
+- CertificateAuthorityActivation
+
+## Getting Started
+
+### Pricing
+Learn more about [AWS Private Certificate Authority Pricing][pricing].
+
+[pricing]: https://aws.amazon.com/private-ca/pricing
```

Comment:
This is miss leading, ACK controllers are open source and free to use. Maybe just include a link to ACM documentation